package/kernel/linux/modules/multimedia.mk

```diff
@@ -0,0 +1,115 @@
+MMEDIA_MENU:=Multimedia
+
+define KernelPackage/multimedia
+  SUBMENU:=$(MMEDIA_MENU)
+  TITLE:=Multimedia
+  KCONFIG:= \
+	CONFIG_MEDIA_SUPPORT=m \
```

Comment:
`CONFIG_MEDIA_SUPPORT` gets already activated by kmod-video-core in `package/kernel/linux/modules/video.mk`

maybe `kmod-video-core` should later depend on `kmod-multimedia`

Reply:
Yes, I see that Video menu includes both output (framebuffer graphics, backlight) and input (cameras).
I could move all input devices into my Multimedia menu, just like the kernel menuconfig is organised.
Is that OK?

Reply:
Yes that sounds good

Reply:
Done.